assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -137,7 +141,14 @@ const StoryGridView = ({
             <DetailRow>
               <CardTitle
                 title={story.title}
-                modifiedDate={story.modified.startOf('day').fromNow()}
+                author={users[story.author].name}
+                status={story.status}
+                // this is fine according to the i18n package: https://developer.wordpress.com/themes/i18n/
```

Comment:
No it's not 🙂 

a) That's not the package we're using (we use `@wordpress/i18n` and other WordPress.org things, nothing from WordPress.com)
b) That link is for PHP only.

Reply:
:) thanks for letting me know i was poling docs from the wrong type of wordpress resource. I will update accordingly.